docs/source/user_guide/data_checks.ipynb

```diff
@@ -304,6 +305,48 @@
     "    print(\"Error:\", message['message'])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Uniqueness Data Check\n",
+    "The `UniquenessDataCheck` is used to detect columns with either too unique or not unique enough values.  For regression type problems, the data is checked for a lower limit of uniqueness.  For multiclass type problems, the data is checked for an upper limit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
```

Comment:
I think we're leaving these cells unrun so that they can run during the doc builds.